src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -24,6 +24,27 @@
 AuthEntity = NewType('AuthEntity', str)
 
 
+def get_auth_entity(daemon_type: str, daemon_id: str, host: str = "") -> AuthEntity:
```

Comment:
we should maybe change this to `_get_auth_entity` or something. We already have another function called `get_auth_entity` (that's calling this one) within the CephadmDaemonDeploySpec class which could be confusing.